### DISCUSSION TITLE
Welcome to vite-react-static-seo Discussions!

### DISCUSSION THREAD

**Discussion by hunghg2505** [hunghg2505/vite-react-static-seo#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hunghg255** [comment]
Nice

**Reply by hunghg2505** [reply]
OK

**Reply by hunghg255** [reply]
OK
